libraries/api/test/logging_test.js

```diff
@@ -237,4 +259,112 @@ suite(testing.suiteName(), function() {
       Logger: 'taskcluster.lib-api',
     });
   });
+
+  test('bewit is elided', async function() {
+    const url = libUrls.api(helper.rootUrl, 'test', 'v1', '/bewitiful?bewit=abc123&foo=abc');
+    const {header} = hawk.client.header(url, 'GET', {
+      credentials: {id: 'client-with-aa-bb-dd', key: 'ignored', algorithm: 'sha256'},
+    });
+    await request.get(url).set('Authorization', header);
+
+    assert.equal(helper.monitorManager.messages.length, 1);
+    delete helper.monitorManager.messages[0].Fields.duration;
+    delete helper.monitorManager.messages[0].Fields.expires;
+    assert.deepEqual(helper.monitorManager.messages[0], {
+      Type: 'monitor.apiMethod',
+      Severity: LEVELS.notice,
+      Fields: {
+        name: 'bewitiful',
+        apiVersion: 'v1',
+        clientId: '',
+        hasAuthed: false,
+        method: 'GET',
+        public: true,
+        query: {
+          foo: 'abc',
+          bewit: '...',
+        },
+        resource: '/api/test/v1/bewitiful',
```

Comment:
@djmitche actually, while I'm looking at it, we can remove `baseUrl` from here to make this just `/bewitful` to save some bytes since we already have `apiVersion` in the logging method and the rest of this string is useless. Does that make sense to you?

Reply:
Sounds OK to me.  That should be noted in the changelog :)